.github/workflows/ci.yml

```diff
@@ -55,9 +55,13 @@ jobs:
     # Build debian packages
     strategy:
       matrix:
-        suite: [jammy]
+        suite: [jammy, bullseye, bookworm]
         include:
           - suite: jammy
+            os-version: ubuntu-20.04
```

Comment:
I sitll have a bad feeling about this, and the same release step around line 460.
[They are deprecating 20.04 GHA runners](https://github.com/actions/runner-images/issues/11101) so this will no longer work.
I wonder, can we just run everything on 22.04?

Reply:
Debian package build and installation tests run within a container build from the according os versions as declared within the docker-compose.yml. That's IMO independent of what the GHA runner is built on, but yes, the runners should also be updated. As the bullseye and bookworm builders are run on ubuntu-22.04 GHA runner, that will most probably fit for jammy too.